src/main/java/mdlaf/components/menu/MaterialMenuUI.java

```diff
@@ -65,7 +68,7 @@ public void uninstallUI(JComponent c) {
 
 	@Override
 	public void paint (Graphics g, JComponent c) {
-		super.paint (g, c);
+		super.paint (MaterialDrawingUtils.getAliasedGraphics (g), c);
```

Comment:
This is not necessary because MaterialMenuUI it is a fake component and under the hood is a MateriaMenuItem, and the library support this function already.

The superclass of MaterialMenuUI contains the following code

```
    public void paint(Graphics g, JComponent c) {
        paintMenuItem(g, c, checkIcon, arrowIcon,
                      selectionBackground, selectionForeground,
                      defaultTextIconGap);
    }
```